cli/src/commands/sidekick.rs

```diff
@@ -0,0 +1,197 @@
+use anyhow::{Context, Result};
+use clap::Parser;
+use cli_core::{
+	ctx,
+	rivet_api::{self},
+};
+use console::Term;
+use serde::Serialize;
+
+use anyhow::bail;
```

Comment:
Nit: merge in to import on line 1

Reply:
Resolved in 4d8e35812bfcaa6930469aea2c9f7ad4242b30a9